support-frontend/assets/pages/supporter-plus-landing/components/PaymentIcons.tsx

```diff
@@ -0,0 +1,114 @@
+import { css } from '@emotion/react';
+import { space } from '@guardian/source/foundations';
+
+const paymentMethods = css`
+	display: flex;
+	margin-left: ${space[1]}px;
+	gap: ${space[1]}px;
+	align-self: center;
+
+	> svg {
+		height: 1.3rem;
+	}
+`;
+
+export function PaymentCards(): JSX.Element {
```

Comment:
Having the component return type is a little bit of a hangover, but doesn't really cause any fuss. 

There's [some good docs here explaining why](https://github.com/typescript-cheatsheets/react#function-components) - and is [adhered to in places like `dotcom`](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/.eslintrc.js#L249-L259).